recipe/meta.yaml

```diff
@@ -27,16 +26,15 @@ requirements:
     - {{ cdt('libxext') }}           # [linux]
   host:
     - python
-    - numpy =1.16  # [py < 39]
-    - numpy =1.19  # [py >= 39]
+    - numpy
     - pip
     - glew
     - libglu                         # [linux]
     - libpng
     - freetype
     - glm
     - xorg-libxfixes
-    - libnetcdf =4.7.4
+    - libnetcdf
```

Comment:
This is similar to the `numpy` pinning. There is a global pin that is currently set to 4.8.1
```
libnetcdf:
  - 4.8.1
```
in https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml. If 4.7.4 is compatible with 4.8.1, then I can add back the earlier version. If not, then there might be conflicts if someone installs another package with a `libnetcdf` dependency that is built against 4.8.1.
Generally, I would follow the global pinnings since that would help avoid compatibility issues with other packages from conda-forge.

Reply:
This sounds reasonable to me